### DISCUSSION TITLE
Add switch CarPlay\iPhone display button

### DISCUSSION THREAD

**Discussion by k-koleda** [organicmaps/organicmaps#4133] [Ideas]
well. from time to time in much easier to setup route or add point or remove point from mobile instead of CarPlay display. now I need to turn off CarPlay or disconnect mobile from car to be able to edit my route. to be honest it really annoying especially when you drive. it would be great to have switch button on mobile something like "you connected to CarPlay would you like to continue on mobile?"

**Comment by biodranik** [comment]
That's a big and important issue. A proper fix of it requires a full refactoring of our rendering engine to properly display maps on both devices. Adding temporary workarounds may not be worth it, but should be investigated too. The plan is to implement Android Auto in the right way, and then reuse a similar approach to fix the CarPlay issue.
